### PR TITLE
update mainnet info json

### PR DESCRIPTION
**Pull request by manoj9april**

### CODE REVIEW

**Review by arpan-jain — CHANGES_REQUESTED** [review]
Can you remove the indentation changes? I am not able to see actual changes because of that.

File: mainnet-deployment-info.json [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+	"eth_proxy_admin": "0x6CBd89A4919E39Ad4c7718B04443CC1722B2cB2A",
+	"eth_maticX_proxy": "0xf03A7Eb46d01d9EcAA104558C732Cf82f6B6B645",
+	"eth_maticX_impl": "0x5a78f4BD60C92FCbbf1C941Bc1136491D2896b35",
+	"eth_validator_registry_proxy": "0xf556442D5B77A4B0252630E15d8BbE2160870d77",
+	"eth_validator_registry_impl": "0x0a7f554AbFBD710D9CDFB7Cf88217a91acA7457D",
+	"rate_provider": "0xeE652bbF72689AA59F0B8F981c9c90e2A8Af8d8f",
+	"polygon_fx_state_child_tunnel": "0x97E58a6950D86d751082D5e1d350e74c19047570",
+	"eth_fx_state_root_tunnel": "0x40FB804Cc07302b89EC16a9f8d040506f64dFe29",
+	"polygon_multisig": "0x51358004cFe135E64453d7F6a0dC433CAba09A2a",
+	"polygon_proxy_admin": "0x79f2Ec80e2B9fB995D9deBda4d9026eF7A1507b9",
```
Comment (galacticminter) [inline mainnet-deployment-info.json:17]: polygon_child_pool_proxy_admin

**Comment by manoj9april** [comment]
> Can you remove the indentation changes? I am not able to see actual changes because of that.
@arpan-jain 
done

File: mainnet-deployment-info.json [hunk 2]
```diff
@@ -4,14 +4,17 @@
   "treasury": "0x80A43dd35382C4919991C5Bca7f46Dd24Fde4C67",
   "matic_erc20_address": "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0",
   "matic_stake_manager_proxy": "0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908",
-  "proxy_admin": "0x80A43dd35382C4919991C5Bca7f46Dd24Fde4C67",
-  "maticX_proxy": "0xf03A7Eb46d01d9EcAA104558C732Cf82f6B6B645",
-  "maticX_impl": "0x648cf74330da142D4e774B14D6C79072c8Cc8701",
-  "validator_registry_proxy": "0xf556442D5B77A4B0252630E15d8BbE2160870d77",
-  "validator_registry_impl": "0x0a7f554AbFBD710D9CDFB7Cf88217a91acA7457D",
+  "eth_multisig": "0x80A43dd35382C4919991C5Bca7f46Dd24Fde4C67",
+  "eth_proxy_admin": "0x6CBd89A4919E39Ad4c7718B04443CC1722B2cB2A",
```
Comment (galacticminter) [inline mainnet-deployment-info.json:8]: eth_maticx_proxy_admin

Comment (manoj9april) [reply mainnet-deployment-info.json:8]: it is also the proxy admin of validator_registry

Comment (manoj9april) [reply mainnet-deployment-info.json:8]: it is also the proxy admin of validator_registry

File: mainnet-deployment-info.json (hunk 1 above)
Comment (manoj9april) [reply mainnet-deployment-info.json]: please check my comment on eth_proxy_admin

**Review by arpan-jain — APPROVED** [review]

**Review by dulguun-staderlabs — APPROVED** [review]
Looks good 👍

**Pull request merged**